static/styles/popovers.css

```diff
@@ -53,43 +53,100 @@
     float: none;
 }
 
-ul.sender_info_popover i {
-    display: inline-block;
-    width: 14px;
-    text-align: center;
-    margin-right: 3px;
+.message-info-popover {
+    padding: 0;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+
 }
 
-ul.actions_popover i {
-    display: inline-block;
-    width: 14px;
-    text-align: center;
-    margin-right: 3px;
+.message-info-popover .popover_info {
+    padding: 5px 10px;
+    width: calc(100% - 20px);
+    position: absolute;
+    bottom: 0;
+
+    color: #fff;
+    text-align: left;
 }
 
 .user_popover {
     top: 100px!important;
     margin: -14px;
 }
 
-.popover-avatar .popover-inner {
-    width: 240px;
+.message-info-popover .popover_info .user-full-name {
```

Comment:
Specifying .message-info-popover kind of broke the popover which appears when we click on the user list on the side. The popover uses the class `user_popover` instead of `.message-info-popover`